specs/protocol/configurability.md

```diff
@@ -441,4 +459,11 @@ configuration of the permissioned dispute game.<br/>
 **Requirement:** No requirement<br/>
 **Notes:** <br/>
 
+### Operator Fee Manager
+
+**Description:** Account authorized to modify the operator fee scalar. <br/>
+**Administrator:** [System Config Owner](#admin-roles)<br/>
```

Comment:
The standard value should be `address(0)` so that they cannot be turned on for standard config chains, otherwise its going to require a lot more work with governance and getting the fee split set up for this for superchain production chains

Reply:
Good with merge once this is updated

Reply:
Fixed